### PR TITLE
Add support to MK320 mouse and keyboard combo, and G700 Gaming Mouse receiver

### PR DESCRIPTION
**Pull request by jhonny-oliveira**

### CODE REVIEW

**Comment by jrbenito** [comment]
Relates to #301

File: rules.d/42-logitech-unify-permissions.rules [hunk 1]
```diff
@@ -33,7 +34,7 @@ DRIVERS=="logitech-djdevice|logitech-hidpp-device", GOTO="solaar_end"
 TAG+="uaccess", TAG+="udev-acl"
 
 # Grant members of the "plugdev" group access to receiver (useful for SSH users)
-#MODE="0660", GROUP="plugdev"
+MODE="0660", GROUP="plugdev"
```
Comment (Lekensteyn) [inline rules.d/42-logitech-unify-permissions.rules]: Please undo this change, it should not be needed for modern distros.

**Comment by muzena** [comment]
@jhonny-oliveira 
@Lekensteyn 

G700 Gaming Mouse receiver

https://github.com/pwr/Solaar/pull/210

**Comment by jhonny-oliveira** [comment]
Ok, done. Do I have to create a new pull request?!...

Best regards,
Jhonny Oliveira

**Comment by jrbenito** [comment]
If you used same branch, New PR is not necessary

Em 10 de nov de 2016 5:40 AM, "Jhonny" <notifications@github.com> escreveu:

> Ok, done. Do I have to create a new pull request?!...
>
> Best regards,
> Jhonny Oliveira
>

**Comment by Lekensteyn** [comment]
I've merged your changes in 9a9a82016fff6cf82e10c3ea0158fd68f2f41724  and squashed the last commit in the previous one. Thanks!
